src/snooping.jl

```diff
@@ -0,0 +1,78 @@
+
+function snoop(path, compilationfile, csv)
+    cd(@__DIR__)
+    # Snoop compiler can't handle the path as a variable, so we just create a file

```

Comment:
I thought I had fixed many of the issues here in SnoopCompile (missing `esc` in macros, need for `tmp_mod`, some cases of bad printing). Are there issues filed for these?

Reply:
I will take a second look! I haven't filed any issues yet